linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_overflow_action/tests/not_present.fail.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+# Ensure test system has proper directories/files for test scenario
+bash -x setup.sh
+
+rm -f $config_file
```

Comment:
```suggestion
#!/bin/bash
# Ensure test system has proper directories/files for test scenario
bash -x setup.sh

config_file=/etc/audit/auditd.conf

sed -i "s/^.*overflow_action.*$//" $config_file
```

Reply:
We're already testing for the the file not being present.. see test `file_not_present.fail.sh`

This one was already good.. remember, we wanted to test a few things:

- [x] Empty file -> `empty.fail.sh`
- [x] File not present at all -> `file_not_present.fail.sh`
- [ ] The overflow_action line not being present -> that's what this test should be